docs/src/pages/guides/interoperability/EmotionCSS.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable react/react-in-jsx-scope -- Unaware of jsxImportSource */
```

Comment:
Let's revert the changes outside of `examples/nextjs-with-styled-components-typescript`